ccvm_simulators/solvers/dl_solver.py

```diff
@@ -666,8 +675,8 @@ def __call__(
                 f"Solver option type {type(algorithm_parameters)} is not supported."
             )
 
-        # Stop the timer for the solve
-        solve_time = time.time() - solve_time_start
+        # Stop the timer for the solve to compute the solution time for solving an instance once
+        solve_time = (time.time() - solve_time_start) / batch_size
```

Comment:
It would also be helpful to clarify what solve_time represents in the other places we have documented it:
- [in Solution](https://github.com/1QB-Information-Technologies/ccvm/blob/dev/solvers_parameters_update/ccvm_simulators/solution.py#L18)
- [in README](https://github.com/1QB-Information-Technologies/ccvm/blob/dev/solvers_parameters_update/README.md?plain=1#L125)

Reply:
Very good point @KatieO4t1qbit , and thanks for your suggestion. This is something we have extensively discussed and have come to the conclusion that this should be the way we measure solving time when using `gpu`. We are comparing different hardware and the advantage of a GPU lies in doing the parallelization. Since GPU can solve multiple instances simultaneously, it is faster to use to solve an instance once compare to CPU. 

I will add a line to the comment above this line and update the documentations in `Solution` and `README`. Thanks!

Reply:
Thanks, Farhad! Looks good.